frontend/apps/reader/modules/readergesture.lua

```diff
@@ -838,6 +857,14 @@ function ReaderGesture:setupGesture(ges, action)
         ratio_x = 7/8, ratio_y = 1/8,
         ratio_w = 1/8, ratio_h = 7/8,
     }
+    local zone_top_edge = {
+        ratio_x = 1/8, ratio_y = 0,
+        ratio_w = 7/8, ratio_h = 1/8,
+    }
+    local zone_bottom_edge = {
+        ratio_x = 1/8, ratio_y = 7/8,
+        ratio_w = 7/8, ratio_h = 1/8,
+    }
```

Comment:
I guess these `1/8` and `7/8` are to exclude the `zone_top_left_corner` (and friends) which are themselves all `1/8` rectangles at corners.
So, I guess this `7/8` must be `6/8`, to leave `1/8` on each side.
Or may be it doesn't matter as there is no conflict with tap/hold on corners, as it is swipe.
And there may be no conflict with each other edge swipes because of different directions.
In which case there is no reason to not use the full width.
So, either an offset of 1/8 and a lenght of 6/8 - or an offset of 0 and a length of 1 (8/8).

(I thought that for consistency, these could also be defined in defaults.lua like the corners' ones are, eg. `DTAP_ZONE_TOP_LEFT`, but I guess there's no reason to override them, so no need to bother.)

Reply:
> Or may be it doesn't matter as there is no conflict with tap/hold on corners, as it is swipe.

You can imagine swipe from corner though. Not sure if it's worth it. :-)

> I guess there's no reason to override them, so no need to bother.)

I don't know, it's kind of cool to hack touch zones to do whatever you want.

Reply:
Zones was simply taken from kobolight plugin :)
https://github.com/koreader/koreader/commit/c6c8c5a0374e89f6a363f2ba9e78834929db3a04#diff-9cf37ee7352547f3ecc054072ae290d6L20

There is no conflict with tap/hold on corners but I don't think that swipe from one corner to another is useful :)

Reply:
OK, but if they were messy there in the plugin, we could fix them now that they are in the core :)

> I don't think that swipe from one corner to another is useful 

If I read them right, they set the following zones:
![image](https://user-images.githubusercontent.com/24273478/62477601-9ba2bb00-b7a9-11e9-8427-260aa8cc4557.png)
and if I remember correctly, it's the swipe _starting position_ that matters for checking the zone.

So, a right handed person holding the device with his single right hand could bottom swipe left with his thumb (starting in the bottom right corner) with no issue.
While a left handed person holding the device with his single left hand will have a hard time as it can't start in the bottom left corner, and will have to extend his thumb far right to reach the green zone.
So, I guess for consistency and the various aerodynamics of hands and fingers, they should all be 8/8 (=1) long :) (if that cause no other issues I have not foreseen.)

Reply:
I'm absolutely convinced :)
I prepare patch for it.